cmake/modules/BuildPlugin.cmake

```diff
@@ -63,3 +63,63 @@ MACRO(BUILD_PLUGIN PLUGIN_NAME)
 	SET_DIRECTORY_PROPERTIES(PROPERTIES ADDITIONAL_MAKE_CLEAN_FILES "${ER_H} ${plugin_MOC_out}")
 ENDMACRO(BUILD_PLUGIN)
 
+MACRO(BUILD_LIKE_A_PLUGIN PLUGIN_NAME)
+  CMAKE_PARSE_ARGUMENTS(PLUGIN "" "" "MOCFILES;EMBEDDED_RESOURCES;UICFILES;JUSTALIBRARY" ${ARGN})
```

Comment:
What is `JUSTALIBRARY`?  I'm having a hard time finding documentation on this...